package.json

```diff
@@ -28,14 +28,12 @@
   "license": "ISC",
   "dependencies": {
     "@babel/runtime": "^7.4.3",
-    "@testing-library/dom": "^7.0.4",
-    "simmerjs": "^0.5.6"
+    "@testing-library/dom": "^8.17.1"
```

Comment:
Since the major version number of `@testing-library/dom` has changed it probably has breaking changes, so we should also bump our major version number also.

I don't really want to do that in this PR but if you raise a new one with this change I'll happily do that 👍